techniques/aria/ARIA17.html

```diff
@@ -2,95 +2,79 @@
             </p>
-            <p>For a group of radio buttons, one could also use role="radiogroup" instead of role="group".</p>
-            <p>The group can be labeled using <code class="att">aria-labelledby</code>.
+            <p>ラジオボタングループの場合、role="group" の代わりに role="radiogroup" を使用できる。
             </p>
-            <p>This technique is not meant for wrapping  all controls on a form within a single container
-               with role="group".
+            <p>グループは <code class="att">aria-labelledby</code> を使用してラベル付けできる。
+            </p>
+            <p>この達成方法は、role="group" をもつ単一のコンテナ内にフォーム上のすべてのコントロールを包むためのものではない。
             </p>
          </section>
          <section id="examples">
-            <h2>Examples</h2>
+            <h2>事例</h2>
             <section class="example" id="example-1-social-security-number">
-               <h3>Example 1: Social Security Number</h3>
-               <p>Social security number fields which are 9 digits long and broken up into 3 segments
-                  can be grouped using role="group".
+               <h3>事例 1: 社会保障番号</h3>
+               <p>9 桁の長さで、三つのセグメントに分割される社会保障番号フィールドは、role="group" を使用してグループ化できる。
                </p><pre xml:space="preserve">&lt;div role="group" aria-labelledby="ssn1"&gt;
    &lt;span id="ssn1"&gt;Social Security#&lt;/span&gt; 
    &lt;span style="color: #D90D0D;"&gt; * &lt;/span&gt;
    &lt;input size="3" type="text" aria-required="true" title="First 3 digits" /&gt;-
    &lt;input size="2" type="text" aria-required="true" title="Next 2 digits" /&gt;-
    &lt;input size="4" type="text" aria-required="true" title="Last 4 digits" /&gt;
-&lt;/div&gt;</pre><p class="working-example">Working example: <a href="../../working-examples/aria-grouping-multipart-fields-ssn/">Multiple part field groups</a>.
+&lt;/div&gt;</pre><p class="working-example">実装例: <a href="https://www.w3.org/WAI/WCAG21/working-examples/aria-grouping-multipart-fields-ssn/">複数パートに分かれたフィールドのグループ化</a>.
                </p>
             </section>
             <section class="example" id="example-2-identifying-radio-groups">
-               <h3>Example 2: Identifying radio groups</h3>
-               <p>This example demonstrates use role=radiogroup. Note also that the radio buttons are
-                  custom controls with role=radio. (But the script to make the span actually work like
-                  radio buttons is not included in this example. ) One may optionally employ CSS to
-                  place a border around groups of such fields to visually reinforce the group relationship.
-                  The CSS properties are available below the form.
+               <h3>事例 2: ラジオグループを識別する</h3>
```

Comment:
```suggestion
               <h3>事例 2: ラジオグループを特定する</h3>
```

> タイトルと事例 2の実装例が英文は同じなのに、翻訳文が違いますけど問題ありませんでしょうか？

WG4用語集のとおり、特定にあわせておきます